solr/core/src/java/org/apache/solr/core/PluginBag.java

```diff
@@ -27,8 +27,7 @@
 import org.apache.solr.common.util.StrUtils;
 import org.apache.solr.handler.RequestHandlerBase;
 import org.apache.solr.handler.component.SearchComponent;
-import org.apache.solr.jersey.CoreContainerApp;
-import org.apache.solr.jersey.SolrCoreApp;
+import org.apache.solr.jersey.JerseyApplications;
```

Comment:
Why plural?   Is this because it hosts mulitple applications in Jersey?

Reply:
Yep!

An "Application" in JAX-RS is pretty far from the colloquial usage.  It's not a whole process or package or something runnable - it's more like Solr's existing "PluginBag" abstraction - a collection of endpoints and config needed to look them up and invoke them.

"JerseyApplications" is plural because it holds multiple of these application configs.  There are two currently: `CoreContainerApp` is a registration point for our container/cluster level APIs, and `SolrCoreApp` handles per-core API registration.